sql/create_users_and_user_roles.sql

```diff
@@ -0,0 +1,92 @@
+SET @nursesalt = (SHA2(SUBSTRING(MD5(RAND()), -10), 512));
+SET @doctorsalt = (SHA2(SUBSTRING(MD5(RAND()), -10), 512));
+SET @receptionsalt = (SHA2(SUBSTRING(MD5(RAND()), -10), 512));
```

Comment:
@prashantsable1991 please add line breaks between queries for a better readability